### PR TITLE
os/tools/download: add README for download functionlity

### PR DESCRIPTION
**Pull request by sunghan-chang**
This document shows what it is and how to use.

Signed-off-by: sunghan-chang <sh924.chang@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [311d6c3263715e22c3ab3ed85ea5d1107f451d9f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [311d6c3263715e22c3ab3ed85ea5d1107f451d9f] - Code Rule Check (C++) OK.

File: os/tools/README_download.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Zipdownload / Download
+
+This guides how to use *zipdownload.sh* and *download.sh* / *download.bat*.  
+They supports binaries programming without TizenRT source code.
```
Comment (jeongchanKim) [inline os/tools/README_download.md]: supports -> support

**Comment by seinfra** [comment]
Target : [d63a5e4b60dd5d1a92f4c826b779d60fc9db5783] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [7aa10a56faa8786fb1a4c97e503e26a3eebbc3d4] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7aa10a56faa8786fb1a4c97e503e26a3eebbc3d4] - Code Rule Check OK.

File: os/tools/README_download.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Zipdownload / Download
+
+This guides how to use *zipdownload.sh* and *download.sh* / *download.bat*.  
+They provide binaries programming without TizenRT source codes.  
+This is useful who does not want to get TizenRT source codes and only wants to program given TizenRT binaries.
+
+## Binary Maker
+
+After building TizenRT, *zipdownload* script zips mandatory files including binaries, download scripts and tools.
```
Comment (dr-venkman) [inline os/tools/README_download.md]: "After building" seems to contradict the claim that the user does not have to build. Please check and ignore if this is not the case.

**Comment by seinfra** [comment]
Target : [f7b034bf9bdad539179e29106d4d3b0698ff2b5b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [4b7f68dde9c4e8fcf946ce7ebfa2299f975fa8f7] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4b7f68dde9c4e8fcf946ce7ebfa2299f975fa8f7] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [4b7f68dde9c4e8fcf946ce7ebfa2299f975fa8f7] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4b7f68dde9c4e8fcf946ce7ebfa2299f975fa8f7] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [4b7f68dde9c4e8fcf946ce7ebfa2299f975fa8f7] - Code Rule Check (C++) OK.

**Pull request merged**